### ISSUE TITLE
Left side panel flashes when paragraphs below menu items are removed.

### ISSUE THREAD

**Issue by margarettanzosh** [framework7io/framework7#2180]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: EXACT RELEASE VERSION OR COMMIT HASH, HERE.
* Platform and Target: PLATFORM CLIENT YOU ARE TARGETING SUCH AS CORDOVA, IOS, ANDROID, ETC.
* Live Link or JSFiddle: PREFERABLY _(IF YOU WANT YOUR ISSUE TO BE RESOLVED ASAP)_. You can start from forking the following JSFiddle template: [JSFiddle](https://jsfiddle.net/nolimits4web/o2ejupu1/)

### What you did
EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.

### Expected Behavior
EXPLAIN WHAT IS TO BE EXPECTED, HERE.

### Actual Behavior
EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.

P.S. Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/framework7)
for that or join the #framework7 channel on irc.freenode.net, where we will be more
than happy to help answer your questions.

Before you open an issue, please check if a similar issue already exists or has been closed before.

**Comment by margarettanzosh** [comment]
To recreate this I used your v2 kitchen sink template and removed the text below the menu items. I also found that the navbar was not appearing properly when list items were removed from the home page. These are both reproduced in the attached code. I previewed this in ios using phonegap.

**Comment by margarettanzosh** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no, viewport-fit=cover">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="default">
  <meta name="theme-color" content="#2196f3">
  <meta http-equiv="Content-Security-Policy" content="default-src * 'self' 'unsafe-inline' 'unsafe-eval' data: gap:">
  <title>Framework7</title>
  <link rel="stylesheet" href="dist/css/framework7.min.css">
  <link rel="stylesheet" href="css/app.css">
  <link rel="apple-touch-icon" href="img/f7-icon-square.png">
  <link rel="icon" href="img/f7-icon.png">
</head>
<body>
  <div id="app">
    <div class="statusbar"></div>
    <div class="panel panel-left panel-cover">
      <div class="page">
        <div class="page-content">
          <div class="block-title">Left Panel</div>
          <div class="block">
            <p>This is a left side panel. You can close it by clicking outsite or on this link: <a href="#" class="panel-close">close me</a>. You can put here anything, even another isolated view like in <a href="#" data-panel="right" class="panel-open">Right Panel</a></p>
          </div>
          <div class="block-title">Main View Navigation</div>
          <div class="list links-list">
            <ul>
              <li>
                <a href="/accordion/" class="panel-close">Accordion</a>
              </li>
              <li>
                <a href="/action-sheet/" class="panel-close">Action Sheet</a>
              </li>
              <li>
                <a href="/badge/" class="panel-close">Badge</a>
              </li>
              <li>
                <a href="/buttons/" class="panel-close">Buttons</a>
              </li>
              <li>
                <a href="/cards/" class="panel-close">Cards</a>
              </li>
              <li>
                <a href="/checkbox/" class="panel-close">Checkbox</a>
              </li>
              <li>
                <a href="/chips/" class="panel-close">Chips/Tags</a>
              </li>
              <li>
                <a href="/contacts-list/" class="panel-close">Contacts List</a>
              </li>
              <li>
                <a href="/data-table/" class="panel-close">Data Table</a>
              </li>
            </ul>
          </div>
          
          
        </div>
      </div>
    </div>

    <div class="panel panel-right panel-reveal">
      <div class="view view-init view-right" name="right" data-url="/">
        <div class="page">
          <div class="navbar">
            <div class="navbar-inner sliding">
              <div class="title">Right Panel</div>
            </div>
          </div>
          <div class="page-content">
            <div class="block">
              <p>This is a right side panel. You can close it by clicking outsite or on this link: <a href="#" class="panel-close">close me</a>. You can put here anything, even another isolated view.</p>
            </div>
            <div class="block-title">Panel Navigation</div>
            <div class="list links-list">
              <ul>
                <li><a href="/panel-right-1/">Right panel page 1</a></li>
                <li><a href="/panel-right-2/">Right panel page 2</a></li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>

    <div class="view view-main view-init ios-edges" data-url="/">
      <div class="page">
        <div class="navbar">
          <div class="navbar-inner">
            <div class="left">
              <a href="#" class="link icon-only panel-open" data-panel="left">
                <i class="icon f7-icons ios-only">menu</i>
                <i class="icon material-icons md-only">menu</i>
              </a>
            </div>
            <div class="title sliding">Framework7</div>
            <div class="right">
              <a class="link icon-only searchbar-enable" data-searchbar=".searchbar-components">
                <i class="icon f7-icons ios-only">search_strong</i>
                <i class="icon material-icons md-only">search</i>
              </a>
            </div>

            <form data-search-container=".components-list" data-search-in="a" class="searchbar searchbar-expandable searchbar-components searchbar-init">
              <div class="searchbar-inner">
                <div class="searchbar-input-wrap">
                  <input type="search" placeholder="Search components"/>
                  <i class="searchbar-icon"></i>
                  <span class="input-clear-button"></span>
                </div>
                <span class="searchbar-disable-button">Cancel</span>
              </div>
            </form>

          </div>
        </div>

        <div class="page-content">
          
          <div class="block-title searchbar-found">Components</div>
          <div class="list components-list searchbar-found">
            <ul>
              <li>
                <a class="item-content item-link" href="/accordion/">
                  <div class="item-media"><i class="icon icon-f7"></i></div>
                  <div class="item-inner">
                    <div class="item-title">Accordion</div>
                  </div>
                </a>
              </li>
              <li>
                <a class="item-content item-link" href="/action-sheet/">
                  <div class="item-media"><i class="icon icon-f7"></i></div>
                  <div class="item-inner">
                    <div class="item-title">Action Sheet</div>
                  </div>
                </a>
              </li>
              <li>
                <a class="item-content item-link" href="/autocomplete/">
                  <div class="item-media"><i class="icon icon-f7"></i></div>
                  <div class="item-inner">
                    <div class="item-title">Autocomplete</div>
                  </div>
                </a>
              </li>

              
              
            </ul>
          </div>
          <div class="list simple-list searchbar-not-found">
            <ul>
              <li>Nothing found</li>
            </ul>
          </div>
          
          
          
          
        </div>
      </div>
    </div>
  </div>
  <script src="dist/js/framework7.min.js"></script>
  <script src="js/routes.js"></script>
  <script src="js/app.js"></script>
</body>
</html>

**Comment by nolimits4web** [comment]
Can't see/replicate such issue. Please provide more information, fill the issue form correctly, make sure you are using latest versions of f7 and cordova, also try is this issue is replicable with WKWebView?

**Comment by margarettanzosh** [comment]
I’m using the the most recent framework7 v2 kitchen sink with phone gap in a template provided by Timo as the basis for my app. I noticed when I open the left panel on my app (I used a dark theme for the left panel) it first opens as a blank white panel for a second before the black appears. I reproduced this error with your kitchen sink template by commenting out lines 58 through 61.

It only does this on my iPhone using phone gap, not on Chrome’s Dev Tools iOS simulator.

I also had a problem with my status bar after hours of reviewing my code and comparing it to yours, I could not figure out, but I reproduced it in your template by commenting out lines 157 though 567. and 118 through 129. 

        

I corrected it by adding 

padding-top: constant(safe-area-inset-top);  /* iOS 11.0 */
padding-top: env(safe-area-inset-top); /* iOS 11.2 */

to the body style in my css but then after rotating the phone to a horizontal position, it ended up looking like this. I thought perhaps there was a bug causing both of these things to happen because I could not find any code in my app that could account for this.

Thank you very much.

Margaret Tanzosh
